plugins/main/public/components/wz-blank-screen/wz-blank-screen.js

```diff
@@ -16,19 +16,73 @@ import {
   PLUGIN_PLATFORM_WAZUH_DOCUMENTATION_URL_PATH_TROUBLESHOOTING,
   PLUGIN_PLATFORM_URL_GUIDE,
   PLUGIN_PLATFORM_URL_GUIDE_TITLE,
+  UI_LOGGER_LEVELS,
 } from '../../../common/constants';
 import { webDocumentationLink } from '../../../common/services/web_documentation';
+import { AppState } from '../../react-services/app-state';
+import { WzMisc } from '../../factories/misc';
+import { getAngularModule } from '../../kibana-services';
+import { ErrorHandler } from '../../react-services/error-handler';
+import { UI_ERROR_SEVERITIES } from '../../react-services/error-orchestrator/types';
+import { getErrorOrchestrator } from '../../react-services/common-services';
 
 export class WzBlankScreen extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      errorToShow: null,
+    };
+
+    // Get $location forom AngularJS
+    const $injector = getAngularModule().$injector;
+    this.$location = $injector.get('$location');
+
+    // Create instance of WzMisc that stores the error
+    this.wzMisc = new WzMisc();
+  }
+
+  componentDidMount() {
+    AppState.setWzMenu();
+    const catchedError = this.wzMisc.getBlankScr();
+    if (catchedError) {
+      let parsed = null;
+      try {
+        parsed = ErrorHandler.handle(catchedError, '', { silent: true });
+      } catch (error) {
+        const options = {
+          context: `${WzBlankScreen.name}.componentDidMount`,
+          level: UI_LOGGER_LEVELS.ERROR,
+          severity: UI_ERROR_SEVERITIES.UI,
+          error: {
+            error: error,
+            message: error.message || error,
+            title: error.name,
+          },
+        };
+        getErrorOrchestrator().handleError(options);
+      }
+
+      this.setState({ errorToShow: parsed || catchedError });
+      this.wzMisc.setBlankScr(false);
+    } else {
+      this.goOverview();
+    }
+  }
+
+  /**
+   * This navigate to overview
+   */
+  goOverview() {
+    this.$location.path('/overview');
```

Comment:
This redirection must be resolved without using AngularJS.

Reply:
I replaced it with a redirection using the `navigateToApp` exposed by the platform core plugin.